.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "playbooks/roles/geerlingguy.php-versions"]
 	path = playbooks/roles/geerlingguy.php-versions
 	url = https://github.com/geerlingguy/ansible-role-php-versions
+[submodule "playbooks/roles/prometheus_redis_exporter_role"]
```

Comment:
@shimulch, this role installs https://github.com/oliver006/redis_exporter which is a lot more generic than what is needed for this task. So it might be better to use something more specific to monitoring the queues like https://github.com/ashishtiwari1993/redis_queue_length_exporter. Please check and pick a maintained alternative CC @0x29a

Reply:
@lgp171188 I've been using [REDIS_EXPORTER_CHECK_KEYS](https://github.com/open-craft/ansible-playbooks/pull/177/commits/9ee8a704467cefbe0cfe42e05f79a21b7e6a0112#diff-7b58e4e987cec3b7a4d4eeb3c36094d4R11) to export length of a specific key, which is documented [here](https://github.com/oliver006/redis_exporter#command-line-flags). It should do the same task as ``llkey`` implementation of ``redis_queue_length_exporter`` (need to test).  

Also [redis_exporter](https://github.com/oliver006/redis_exporter) is better maintained than [redis_queue_length_exporter](https://github.com/ashishtiwari1993/redis_queue_length_exporter) and documented in [prometheus doc](https://prometheus.io/docs/instrumenting/exporters/#databases).

I don't see any other exporter that only exports lengths other than your shared one. Do you want me to change it?